ear/src/main/application/META-INF/was.policy

```diff
@@ -189,6 +189,8 @@ permission java.security.SecurityPermission "printIdentity";
 
 // required for Log4J stacktrace rendering and JavaAssist PipeLineSessionDebugger proxying
 permission java.lang.RuntimePermission "createSecurityManager";
+// required for Log4J stacktrace rendering 
+permission java.lang.RuntimePermission "accessClassInPackage.com.ibm.CORBA.iiop";
```

Comment:
Waarom is een ibm classe nodig voor log4j?

Reply:
Was inderdaad toch niet nodig